Getting-Started-with-Akka-http-signature.md

```diff
@@ -102,11 +102,11 @@ publ: scala.util.Try[java.security.PublicKey] = Success(
 ```
 The .readPublicKeyFrom() function returns a Try of Public/Private key depending on whether the String given can be parsed back to some valid key. If the operation is successful the value of the original key is assigned to the new value in BigInt format.
 
-The user can then save his keys on his local filesystem by using the following ammonite commands:
+The user can then save his keys on his local filesystem within a .keys directory which we will make use of later in the guide by using the following ammonite commands:
 
 ```scala
-write(wd/"publicKey.pem", RSAKeys.save(pub))
-write(wd/"privateKey.pem", RSAKeys.save(priv))
+write(wd/".keys"/"publicKey.pem", RSAKeys.save(pub))
+write(wd/".keys"/"privateKey.pem", RSAKeys.save(priv))
```

Comment:
no it's not in the wd which could be anywhere, and is usually not in the home directory (eg. I have a `~/Programming/Scala/` dir. ammonite.io has a `home` variable for the home dir.